pages/blog/blogDetail.js

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
+import DateTimeFormat from '../../components/Shared/DateTimeFormat';
+import CommentList from '../../components/comments/commentList';
+import CommentForm from '../../components/comments/commentForm';
+
+const BlogDetail = (props) => {
+    return (
+        <>
+            <div className="section-blog-post">
+                <div className="section-blog-post-header">
+                    <h1>{props.title}</h1>
+                    <div className="post-bottom-panel">
+                        <div>
+                            <div className="group-xl">
+                                {props && props.author ? (
+                                    <div className="post-modern-author">
+                                        <span className="post-icon icon mdi mdi-account"></span>
+                                        <span>
+                                            {`by ${props.author[0].firstName} ${props.author[0].lastName}`}
+                                        </span>
+                                    </div>
+                                ) : (
+                                    ''
+                                )}
+                                <div className="post-modern-time">
+                                    <span className="post-icon icon mdi mdi-calendar-clock"></span>
+                                    <DateTimeFormat value={props.postingDate} format="DateTime" />
+                                </div>
+                            </div>
+                        </div>
+                        <div className="d-none d-md-block">
+                            <div className="group-xl">
+                                <div className="post-modern-views">
+                                    <span className="post-icon icon mdi mdi-eye"></span>
+                                    <span>193</span>
+                                </div>
+                                {props && props.blogComment ? (
+                                    <div className="post-modern-comment">
+                                        <span className="post-icon icon mdi mdi-comment"></span>
+                                        <a href="#comments">{props.blogComment.length}</a>
+                                    </div>
+                                ) : (
+                                    ''
+                                )}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                {/* <!-- BLOG IMAGE --> */}
+                <div className="section-blog-post-figure">
+                    <img src={props.image} alt="" />
```

Comment:
`<DotCMSImage />`?

Reply:
the img path to be rendered (/dA/57ee7cd3-66fc-4f01-9b7e-11e53553c220/image/sloth-1879999_960_720.jpg) is not following the path set on `DotCMSImage `

Reply:
You can use it anyway and get the exact size image you need because if a user upload a 2345345mb image that's what you'll get here.